<launch>
	<arg name="veh" doc="The name of the robot (e.g., autobot01, watchtower02, ...)"/>
    <arg name="robot_type" doc="The type of the robot (e.g., duckiebot, watchtower, ...)"/>
    <arg name="pkg_name" value="multirobot"/>
    <arg name="node_name" default="tf_bridge_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param filename. ex: montreal_loop"/>

    <!-- Subscription -->
    <!-- "~detections": duckietown_msgs/AprilTagDetectionArray. Input tag detections -->

    <!-- Publication -->
    <!--  None  -->

    <group ns="$(arg veh)">
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
            <param name="veh" value="$(arg veh)" />
            <param name="robot_type" value="$(arg robot_type)" />
            <rosparam command="load"
                file="$(find multirobot)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
        </node>
    </group>
</launch>
